src/wp-includes/media.php

```diff
@@ -1076,6 +1076,8 @@ function wp_get_attachment_image( $attachment_id, $size = 'thumbnail', $icon = f
 			}
 		}
 
+		$attr['decoding'] = 'async';
```

Comment:
I suppose the question will be whether this should have a filter like lazy-loading does via `wp_lazy_loading_enabled()`. Should there be a `wp_async_decoding_enabled()` function and filter passed the same args?

Reply:
In contrast with lazy-loading in which `lazy` is omitted for images likely to be in the first viewport, the `decoding=async` attribute would still be added to all images.